src/request.js

```diff
@@ -1,5 +1,20 @@
+/**
+ * Runs the redux-bees api request.
+ * The baseUrl can also be a function which is assessed at runtime.
+ *
+ * @param {String|Function} baseUrl
+ * @param {String} path
+ * @param {Object} options
+ * @return {Promise}
+ */
+
 export default function request(baseUrl, path, options) {
-  return fetch(baseUrl + path, options)
+  let url;
+  if (typeof baseUrl === 'function') {
+    url = baseUrl() + path;
```

Comment:
I've now made this into a one-liner ternary call. Let me know if you feel it's better.